src/meta/src/manager/notification.rs

```diff
@@ -151,6 +151,7 @@ impl NotificationManager {
             WorkerType::Frontend => core_guard.compute_senders.remove(&worker_key),
             WorkerType::ComputeNode => core_guard.frontend_senders.remove(&worker_key),
             WorkerType::Compactor => core_guard.compactor_senders.remove(&worker_key),
+            WorkerType::RiseCtl => None,
             _ => unreachable!(),
```

Comment:
BTW, what's `WorkerType::Generic` for? cc @yezizp2012 @Li0k

Reply:
`WorkerType::Generic` is a generic worker type introduced by @Li0k in order to send notification to all nodes. But it seems like not needed anymore?